parachains/runtimes/assets/statemint/src/lib.rs

```diff
@@ -847,13 +945,14 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
 				// System Events
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+				// Configuration ActiveConfig
+				hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385").to_vec().into(),
```

Comment:
```suggestion
				// ACTIVE_CONFIG
				hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385").to_vec().into(),
```

This is only whitelisted for statemint. Intentional?  
There should also be a constant in `polkadot-primitives` for this, then you dont need to put a magic number here.

Reply:
Needs to be whitelisted for all the asset runtimes, good catch.